Neos.Flow/Classes/Mvc/View/ViewInterface.php

```diff
@@ -59,7 +63,7 @@ public function canRender(ControllerContext $controllerContext);
     /**
      * Renders the view
      *
-     * @return string The rendered view
+     * @return string|ActionResponse|ResponseInterface|StreamInterface|object The rendered result
```

Comment:
the `|object` part scares me a bit :)
wouldn't `string|obect` be sufficient then?

Reply:
Well, those all have distinct handling… for a generic object a check against availalability of `__toString()` is done.

Reply:
Fair enough